Moggles.Data.SQL/SQLFeatureToggleStatus.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Moggles.Data.SQL
+{
+    public class SQLFeatureToggleStatus
+    {
+        public int Id { get; set; }
+        public bool Enabled { get; set; }
+        public bool IsDeployed { get; set; }
+        public DateTime? FirstTimeDeployDate { get; set; }
+        public DateTime? LastDeployStatusUpdate { get; set; }
+
+        public SQLDeployEnvironment Environment { get; set; }
+        public int EnvironmentId { get; set; }
+
+        public SQLFeatureToggle FeatureToggle { get; set; }
+        public int FeatureToggleId { get; set; }
+        public Guid NewId { get; set; } = Guid.NewGuid();
```

Comment:
Some of these properties aren't being used (featuretogglestatus.newid and EnvironmentID)